eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -147,7 +147,7 @@ stages:
           - ${{if ne(artifact.options.skipPublishDocs, 'true')}}:
```

Comment:
```suggestion
          - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
```

Reply:
Do we want to control this by DocMs only? GithubIO is still using `options.skipPublishDocs`. Then we have two vars for two doc steps.

Reply:
I'd change the github.io publishing `skipPublishDocGithubIo` and we probably should remove the "options." from all these conditions.